test/attr/attr_inlinable_available.swift

```diff
@@ -1356,7 +1356,7 @@ extension BetweenTargets {
 
 @available(macOS 10.10, *)
 extension BetweenTargets { // expected-error {{'BetweenTargets' is only available in macOS 10.14.5 or newer; clients of 'Test' may have a lower deployment target}}
-  public func publicFuncInExcessivelyAvailableExtension() {}
+  public func publicFuncInExcessivelyAvailableExtension() {} // expected-note {{instance method 'publicFuncInExcessivelyAvailableExtension()' declared here must be visible to other modules}}
```

Comment:
I'm finding this note to be confusing in this context. Its connection to the original diagnostic (`'BetweenTargets' is only available...`) is not clear, IMO.

Reply:
(I mean, the connection is clear to me as an implementer of availability diagnostics, but I'm trying to put myself in the shoes of an average developer).